tests/unit/api/test_metadata.py

```diff
@@ -339,11 +357,18 @@ def test_post_metadata_sign_delete(self, test_client, monkeypatch):
             mocked_repository_metadata,
         )
         payload = {"role": "root"}
+        fake_time = datetime.datetime(2019, 6, 16, 9, 5, 1)
+        fake_datetime = pretend.stub(
+            now=pretend.call_recorder(lambda: fake_time)
+        )
```

Comment:
At the end of the test, you want to add a line testing the `fake_datetime.now` calls.
Check how we assert calls for other call recorders in this function.